### DISCUSSION TITLE
Unable to open CodeSandbox examples

### DISCUSSION THREAD

**Discussion by alaqsaka** [igordanchenko/yet-another-react-lightbox#224] [Q&A] [answered]
Hi, I'm interested in seeing an example for Carousel. When I click open code in sandbox, the result is like this, has anyone experienced the same thing? Is this an error from me or from the owner of this project in sandbox? Thanks! 
![image](https://github.com/igordanchenko/yet-another-react-lightbox/assets/40936981/507dc337-fe7e-4c5f-85ed-be922a2b99cd)

**Comment by igordanchenko** [comment, marked as answer]
It was a CodeSandbox glitch, and the sandbox VM required a reboot. The sandbox is working correctly now. Thank you for alerting me about this issue.

**Reply by alaqsaka** [reply]
Yup, I emailed the CodeSandbox team and they said it there was as an error, and now it's working fine, thanks!
